apps/condo/domains/marketplace/components/MarketplacePageContent/ContextProvider.ts

```diff
@@ -1,17 +1,17 @@
 import { createContext, useContext } from 'react'
 
-import type { InvoiceContext as InvoiceContextSchema } from '@app/condo/schema'
+import type { AcquiringIntegrationContext as AcquiringIntegrationContextSchema } from '@app/condo/schema'
 
-type IInvoiceContext = {
-    invoiceContext: InvoiceContextSchema
-    refetchInvoiceContext: () => void
+type IAcquiringContext = {
+    acquiringContext: AcquiringIntegrationContextSchema
+    refetchAcquiringContext: () => void
 }
 
-export const InvoiceContext = createContext<IInvoiceContext>({
-    invoiceContext: null,
-    refetchInvoiceContext: () => ({}),
+export const AcquiringContext = createContext<IAcquiringContext>({
```

Comment:
3rd same context in single PR? Why not just fetch it?

Reply:
Only this one was kept for marketplace pages.

Reply:
Why do you need a separate context for that. There's no difference in using it via `useObjects`

Reply:
It's more convinient. At least because I need to set `where` only once.